README.md

```diff
@@ -1,2 +1,63 @@
-# Restaurant-page
+# Restaurant-page (Oslo-diner)
 I used what we’ve learned and take a chance to continue practicing DOM manipulation by dynamically rendering a simple restaurant homepage! At the end, we use use JavaScript alone to generate the entire contents of the website!
+ 
+
+![screenshot](./screenshot.png)
+
+- The application describes what Oslo diner is all about
+- The application displays all menu of food available
+- The appplication allows users to fill the contact form
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+1.  Clone this repository with
+    `git clone https://github.com/Haroonabdulrazaq/Restaurant-page.git` using your terminal or command line.
+2.  Change to the project directory by entering `cd Restaurant-page` in the terminal
+3.  Next open the folder with an IDE like VSCode.
+4.  Install the LiveServer Extension on VSCode.
+5.  Once installed hit the "Go Live" button on the bottom right hand of your editor.
+6.  The application should automatically open in a browser window.
+7.  Watch the file with:
+   - <code> npm init </code> To install all package dependencies.
+   - <code> npm run build </code> To run webconfig script
+   - <code> ./node_modules/.bin/webpack --watch </code> To watch files
+
```

Comment:
## Add a live link to your page.
- [ ] Kindly add a live link to where your page is hosted live. Let's ensure the readme is as descriptive as possible. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Getting a live link is optional for this project